pkg/model/crd.go

```diff
@@ -303,6 +303,20 @@ func (r *CRD) IsSecretField(path string) bool {
 	return false
 }
 
+// GetImmutableFields returns list of immutable fields present in CRD
+func (r *CRD) GetImmutableFields() []string {
+	fConfigs := r.cfg.ResourceFields(r.Names.Original)
+	var immutableFields []string
+
+	for field,fieldConfig := range fConfigs {
```

Comment:
nit: need a space between `field,` and `fieldConfig` above. (This is usually fixed with `go fmt`, so @nmvk please make sure you run `go fmt` before `git commit`'ing :) or use an autoformatter plugin in your text editor or IDE)